pkg/features/versioned_kube_features.go

```diff
@@ -823,7 +823,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	},
 
 	WindowsHostNetwork: {
-		{Version: version.MustParse("1.26"), Default: true, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("1.26"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
@marosset what are the implications of turning this off now that it's been on for two years? also, is sig-windows working on graduating this to beta?

Reply:
Looks like KEP is marked as  `Closed as not planned` https://github.com/kubernetes/enhancements/issues/3503 we should cleanup/remove the flag and switch the one location we check for this flag to not do that 

https://github.com/kubernetes/kubernetes/blob/ec2e0de35a298363872897e5904501b029817af3/pkg/kubelet/kuberuntime/kuberuntime_sandbox.go#L242-L249

Reply:
... I don't know how to reconcile an unplanned incomplete KEP with what what to do with the behavior that is now active in every supported Kubernetes version ... turning *off* that code will now be a behavior change... is that what we want?

Reply:
@liggitt will leave this as follow up to @marosset and sig-windows. I am leaving the flag as-is here

Reply:
I'll drive deprecating this enhancement starting in v1.33